src/types.ts

```diff
@@ -70,3 +65,18 @@ export const defaultAuthoredState: IAuthoredState = {
   water: WaterAmount.Some,
   co2amount: CO2Amount.None,
 };
+
+export const defaultInitialState: Partial<IInteractiveState> = {
```

Comment:
I think if you get rid of the `Partial<>` you can remove the `!` assertions that are forbidden in the `app.tsx` file.